.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: curl -L https://api.github.com/repos/${{ github.repository }}/tarball/${{ github.sha }} > source.tar.gz
```

Comment:
Should we use `github.event.release.tarball_url` instead?

See https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release

Reply:
awesome!

Reply:
```suggestion
      - run: curl -L ${{ github.event.release.tarball_url }} > source.tar.gz
```